newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -319,6 +320,21 @@ export const GameDetailsDialog = ({
                 multiline
                 rows={5}
               />
+              <SelectField
```

Comment:
Does the disabled SelectField look OK?
I'm wondering if we should just display a disabled Textfield in this case (with a mapping from the value to a readable text)